tests/homework_QA_Manager_table.py

```diff
@@ -0,0 +1,71 @@
+import time
+import unittest
+
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+
+
+class EmployeeList(unittest.TestCase):
+    def setUp(self) -> None:
+        browser = webdriver.Chrome(service=Service(executable_path=ChromeDriverManager().install()))
+        browser.get('http://hrm-online.portnov.com/')
+        self.browser = browser
+    def test_employee_list(self):
+        browser = self.browser
+        browser.find_element(By.CSS_SELECTOR, '#txtUsername').send_keys('admin')
+        browser.find_element(By.CSS_SELECTOR, '#txtPassword').send_keys('password')
+        browser.find_element(By.CSS_SELECTOR, '#btnLogin').click()
+        self.assertIn('/pim/viewEmployeeList', browser.current_url)
+        time.sleep(1)
+        welcome_message = browser.find_element(By.ID, 'welcome').text
+        self.assertEqual('Welcome Admin', welcome_message)
+
+        browser.find_element(By.XPATH, "//select[@id='empsearch_job_title']").click()
+        time.sleep(1)
+        job_title_qa = browser.find_element(By.XPATH, '//option[text()="QA Manager"]').text
```

Comment:
This essentially says, find element with text QA Manager, now assert the text is QA Manager.

The assert is redundant, because of the element is found it is already 100% certain it is QA Manager. If it's not found, you will get an Exception and you test will abort so the assert will never execute.

Use a different locator that is not text based to locate elements for which you're asserting the text.